reference/artifacts/types/overview/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: single
+title:  "Types of Artifacts"
+sidebar:
+  nav: reference
+---
+
+While an artifact can reference any remote, deployable resource, there are
+certain types of artifacts that we have first-class support for in the form of:
```

Comment:
"While an artifact can reference any remote, deployable resource, we have first-class support for certain types of artifacts, in the form of..."

[list]